src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -906,21 +908,19 @@ else if (v instanceof ListValue)
             }
         });
 
-        put("silent",(e,v)->{
-           e.setSilent(v.getBoolean());
-        });
+        put("silent",(e,v)-> e.setSilent(v.getBoolean()));
```

Comment:
Why did you compress
```
        put("silent",(e,v)->{
           e.setSilent(v.getBoolean());
        });
```
to `put("silent",(e,v)-> e.setSilent(v.getBoolean()));`? The original is more readable.
(same for next 3 occurrences)

Reply:
not really. I like it better this way. It's also more in keeping with gnembon's other on-liners